### PR TITLE
Bug 1239546 - Sync error No keyFetchToken after OTA. r=michielbdejong…

### PR DESCRIPTION
**Pull request by ferjm**
…,yzen

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f691a6af296288c936c72f2a4eb9b3af546924dd

File: shared/locales/firefox_sync/firefox_sync.en-US.properties [hunk 1]
```diff
@@ -5,6 +5,8 @@ fxsync-collections-passwords=Passwords
 fxsync-empty-account=This account is empty. You need to sync from Firefox, Firefox for Android or Firefox for iOS first and try again here.
 fxsync-error-invalid-account=This account is inactive.
 fxsync-error-invalid-account-explanation=You need to activate Firefox Sync on Desktop or Android first and then try again.
+fxsync-error-no-key-fetch-token=Please try login in again.
```
Comment (michielbdejong) [inline shared/locales/firefox_sync/firefox_sync.en-US.properties]: Nit: login -> logging

File: apps/settings/test/unit/panels/firefox_sync/panel_test.js [hunk 2]
```diff
@@ -842,22 +843,25 @@ suite('Firefox Sync panel >', () => {
...
 
-    test('onsyncchange errored with ' + ERROR_OFFLINE +
-         ' should show alert', done => {
-      subject.onsyncchange({
-        state: 'errored',
-        error: ERROR_OFFLINE
-      });
+    [ERROR_OFFLINE,
+     ERROR_NO_KEY_FETCH_TOKEN].forEach(error => {
+      test('onsyncchange errored with ' + ERROR_OFFLINE +
```
Comment (yzen) [inline apps/settings/test/unit/panels/firefox_sync/panel_test.js]: ERROR_OFFLINE -> error

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b4500a874021e57da9fc4f74421c872e2b1f4e4d

**Pull request merged**